onboard/rocks_catalog_actions.go

```diff
@@ -0,0 +1,85 @@
+package onboard
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/treeverse/lakefs/catalog"
+	"github.com/treeverse/lakefs/catalog/rocks"
+	"github.com/treeverse/lakefs/cmdutils"
+	"github.com/treeverse/lakefs/graveler"
+	"github.com/treeverse/lakefs/logging"
+)
+
+// RocksCatalogRepoActions is in-charge of importing data to lakeFS with Rocks implementation
+type RocksCatalogRepoActions struct {
```

Comment:
I assume this one is temporary as the name and the interface were made to abstract catalog functionality.

Reply:
Yeah but they can't really abstract for the import scenario. We're using the Catalog/cataloger in a whole different way between the two.

Reply:
Guess it will be part of the pulling out the MVCC implementation